conf/messages.en

```diff
@@ -660,3 +682,8 @@ amend-file-upload.help.fileFormat2= .jpg
 amend-file-upload.help.fileFormat3= .pdf
 amend-file-upload.help.fileFormat4= .png
 amend-file-upload.help.fileFormat5= .xls
+
+your.details.checkYourAnswersLabel = Your details
+contact.details.checkYourAnswersLabel = Contact details
+payment.information.checkYourAnswersLabel = Payment information
+importer.details.checkYourAnswersLabel = Importer details
```

Comment:
```suggestion
importer.details.checkYourAnswersLabel = Importer details

```

Reply:
@lmugnai not sure what needs to be fixed here